CollAction/Views/Projects/Details.cshtml

```diff
@@ -206,7 +206,7 @@
                     }
                     else
                     {
-                        <a class="take-part-button @disableCommit" asp-action="Commit" asp-route-id="@Model.Project.Id">Take Part</a>
```

Comment:
Don't we need to pass the project name here?

Reply:
Parameter is bound from navigation and automatically routing injects in in all links for that action, but might be good to included that anyways in case it is not the case in future due to changes.